main.go

```diff
@@ -66,12 +66,21 @@ func main() {
 	}
 
 	service.Messages[grpc.EmptyReq.Name] = grpc.EmptyReq
+	service.Messages[grpc.TransactOptsReq.Name] = grpc.TransactOptsReq
 
 	for _, f := range contractAbi.Methods {
 		var inputArgs []grpc.Argument
 		var outputArgs []grpc.Argument
 		method := grpc.Method{
-			Name: f.Name,
+			Name:  f.Name,
+		}
+
+		if !f.Const {
```

Comment:
Can we add comment saying that "when method is not constant function, it should add transaction options".
Am I right about the comment btw?

Reply:
Yes, OK!